src/metaswitch/ellis/prov_tools/update_user.py

```diff
@@ -48,14 +48,22 @@ def main():
     parser.add_argument("-q", "--quiet", action="store_true", dest="quiet", help="don't display the user")
     parser.add_argument("--hsprov", metavar="IP:PORT", action="store", help="IP address and port of homestead-prov")
     parser.add_argument("--plaintext", action="store_true", help="store password in plaintext")
+    parser.add_argument("--ifc", metavar="iFC-FILE", action="store", dest="ifc_file", help="XML file containing the iFC")
+    parser.add_argument("--prefix", action="store", default="123", dest="twin_prefix", help="twin-prefix (default: 123)")
     parser.add_argument("dns", metavar="<directory-number>[..<directory-number>]")
     parser.add_argument("domain", metavar="<domain>")
-    parser.add_argument("password", metavar="<password>")
+    parser.add_argument("--password", metavar="<password>")
     args = parser.parse_args()
 
     utils.setup_logging()
     settings.HOMESTEAD_URL = args.hsprov or settings.HOMESTEAD_URL
 
+    ifc = None
+    if args.ifc_file:
+        ifc = utils.build_ifc(args.ifc_file, args.domain, args.twin_prefix)
```

Comment:
Presumably this means that if you didn't use the default prefix to create the user, you have to specify it again when you update the user otherwise it will be reset to 123?